lib/devcycle-ruby-server-sdk/localbucketing/local_bucketing.rb

```diff
@@ -201,13 +201,13 @@ def init_event_queue(options)
       @@instance.invoke("initEventQueue", @sdkKeyAddr, options_addr)
     end
 
-    # sig { params(customdata: Hash).returns(NilClass) }
-    # def set_client_custom_data(customdata)
-    #   customdata_json = Oj.dump(customdata)
-    #   customdata_addr = malloc_asc_string(customdata_json)
-    #   @@stack_tracer = @@stack_tracer_raise
-    #   @@instance.invoke("setClientCustomData", customdata_addr)
-    # end
+    sig { params(customdata: Hash).returns(NilClass) }
+    def set_client_custom_data(customdata)
+      customdata_json = Oj.dump(customdata)
+      customdata_addr = malloc_asc_string(customdata_json)
+      @@stack_tracer = @@stack_tracer_raise
+      @@instance.invoke("setClientCustomData", @sdkKeyAddr, customdata_addr)
```

Comment:
I had a feeling it wasn't sorbet :P

Reply:
lol yup - was working on something else and I was like "wait a minute...."